modules/collection/environment/warning.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  config,
+  ...
+}: let
+  inherit (lib.options) mkEnableOption;
+  inherit (lib.modules) mkIf;
+  inherit (lib.lists) optionals;
+  inherit (builtins) all;
```

```suggestion
  inherit (lib.options) mkEnableOption;
  inherit (lib.modules) mkIf;
  inherit (lib.lists) optionals all;
```

Aliased already. One less line :)